core/src/main/java/org/apache/calcite/rex/RexUnaryBiVisitor.java

```diff
@@ -60,6 +60,10 @@ protected R end(RexNode e, R arg) {
     return end(dynamicParam, arg);
   }
 
+  @Override public R visitNamedParam(RexNamedParam namedParam, R arg) {
```

Comment:
TODO: what is this even for?

Reply:
Honestly I don't think it does anything. It seems like this visitor is the "default" for a class that can do more complex modifications. I'm not sure what those are though.